PluginSetup.php

```diff
@@ -0,0 +1,79 @@
+<?php namespace LeanWoocommerceApi;
+
+/**
+ * Main class loader for initializing and  setting up the plugin.
+ *
+ * @since 0.1.0
+ */
+class PluginSetup {
+
+	/**
+	 * Initialise the program after everything is ready.
+	 *
+	 * @since 0.1.0
+	 */
+	public static function init() {
```

Comment:
This one as well is not required if you are going to load the classes via the autoloading from Composer.